openedx/core/djangoapps/oauth_dispatch/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+Root of the oauth_dispatch app.
+
+This app dispatches oauth request processing to django-oauth2-provider (DOP) or
+django-oauth-toolkit (DOT).
```

Comment:
@nasthagiri: 
1. Do you have a strong opinion about how this should read?
2. For dev docs, is this ok, or do we want to have a README.rst and should the __init__.py files just reference the README?

Reply:
@robrap There's already a README.rst file in this django app.